### ISSUE TITLE
ignoring PATH_STATUS frames doesn’t seem wise

### ISSUE THREAD

**Issue by marten-seemann** [quicwg/multipath#212]
> An Endpoint MAY ignore the PATH_STATUS frame if it would make all the paths unavailable in a single connection.

Ignoring frames makes me feel very uncomfortable. This can easily lead to very hard-to-debug situations.

Ideally, abandoning all active paths would be a connection error. However, there’s the corner case where the packet containing the path probing packets for a new path is reordered. I can see two solutions:
1. Actually make it a connection error, and forbid abandoning all paths until a new path has been verified.
2. Start a 3 PTO timer when this situation occurs, to allow for the arrival of the PATH_CHALLENGE frame. If it doesn’t arrive, kill the connection. Problem: The 3 PTO assumes that the latency of the new path is not too different from the existing paths, which might not be a valid assumption.

**Comment by huitema** [comment]
In fact, the plain language definition of "standby" is "ready when you need it". It should be perfectly OK for a node to start using a standby path if no other path is available.

**Comment by mirjak** [comment]
Agreed - path status is just guidance but doesn't close the path. So a sender can just decide to use any of the stand-by paths in that situation but really shouldn't ignore that frame/info entirely.

I created PR #238 to remove this sentence. I also added the following sentence:

"If all available path are marked as "standby", no guidance is provided about
which path should be used preferably, however, the sender might consider
to minimize the number of concurrently used paths."

Is that some we want to say? Or remove the second part? Or just don't have this sentence at all?

**Comment by qdeconinck** [comment]
I would probably drop the second part of the sentence, otherwise the added text sounds fine.

**Comment by kazuho** [comment]
@qdeconinck 
> I would probably drop the second part of the sentence

SGTM.

It might be confusing to suggest minimizing the number of paths without having guidance on how to choose one. But I do not think we can (or need to try to) give good advice.
